content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -53,3 +64,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+
+.. image:: hong_kong/hk-adw.png
+   :alt: Hong Kong ADW Formula
+
+.. note::
+   For 418 compliance, there is no automated allocation of the **Statutory Holiday**
+   entitlement to the employees. As soon as your employees meets the 418 requirements, you may
+   manually allocate the leaves via the **Time-Off** app.
+
+You may generate the following payslips (ensure payslip status is **Done**) to validate the outcome:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Period
+     - Days
+     - Wage
+     - Commission
+     - Total
+     - ADW
+     - Leave Value
+   * - Jan
+     - 31
+     - $20200
+     - $0
+     - $20200
+     - $651.61 ($20200/31)
+     - N/A
+   * - Feb
+     - 28
+     - $20200
+     - $5000
+     - $25200
+     - $769.49 ($45400/59)
+     - N/A
+   * - Mar (1 Day Annual Leave)
+     - 31
+     - $20324.33
+     - $0
+     - $20324.33
+     - $730.27 ($65724.33/90)
+     - $769.49
+   * - Apr (1 Day 80% Sick Leave)
+     - 30
+     - $20117.56
+     - $0
+     - -
+     - -
+     - $584.22 ($730.27*0.8)
+
+Here is an example demonstrating the 713 logic:
+
+- :guilabel:`Jan`: Generate a payslip with a monthly wage of $20200. The **ADW** is always computed on a cumulative basis of the trailing 12-months.
+- :guilabel:`Feb`: Generate a similar payslip but add an **Other Input Type** for the Commission.
+- :guilabel:`Mar`: We will apply for **one** full-paid annual leave in March. The salary compensation for the leave taken is based on ADW thus far.
+
+.. image:: hong_kong/hk-march-713.png
+   :alt: Hong Kong March 713
+
+- :guilabel:`Apr`: We will apply for a 1-day non-full pay leave in April. Since this is a non-full pay leave, the ADW is computed accordingly.
+
+.. image:: hong_kong/hk-apr-713.png
+   :alt: Hong Kong April 713
+
+.. note::
+   The value of ADW is computed in the backend and not be visible to the user.
+
+.. seealso::
+   - `HK 713 Ordinance <https://www.labour.gov.hk/eng/public/wcp/ConciseGuide/Appendix1.pdf>`_
+   - `HK 418 Ordinance <https://www.workstem.com/hk/en/blog/418-regulations/>`_
+
+Generate your reports
+---------------------
+
+Before generating the below reports, remember to setup the following in
+:menuselection:`Settings --> Payroll --> Accounting/HK Localization`.
+
+.. image:: hong_kong/hk-report-setup.png
+   :alt: Hong Kong Payroll Settings
+
+IRD Report
+~~~~~~~~~~
+
+There are a total of **four** IRD reports available:
+
+- :guilabel:`IR56B`: Employer's Return of Remuneration and Pensions
+- :guilabel:`IR56E`: Notification of Commencement of Employment
+- :guilabel:`IR56F`: Notification of Ceasation of Employment (remaining in HK)
+- :guilabel:`IR56G`: Notification of Ceasation of Employment (departing from HK permanently)
+
+Go to :menuselection:`Payroll --> Reporting --> IR56B/E/F/G`:
+
+#. Click on :guilabel:`New`.
+#. Fill in the relevant information for the IRD report.
+#. Click on :guilabel:`Populate` and the :guilabel:`Eligible Employees` smart button will show up.
```

Comment:
```suggestion
#. Click on :guilabel:`Populate` and the :guilabel:`Eligible Employees` smart button appears.
```
try to avoid "will", "will be" etc. No future tense.